code/modules/projectiles/guns/projectile/revolver/deckard.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/projectile/revolver/deckard
 	name = "\"Deckard\" heavy revolver"
-	desc = "A rare, custom-built revolver formerly used on Sol by early Turing Police. Use when there is no time for the Voight-Kampff test. Uses .50 Kurtz."
+	desc = "A rare, custom-built revolver formerly used by Sol and Mar'qua Empire early Turing Police. Use when there is no time for the Voight-Kampff test. Uses .50 Kurtz."
```

Comment:
I'd personally change this to 'A rare, custom-built revolver formerly used within Sol by the early Turing Police, gifted by the Mar'qua Empire. Use when there is no time for the Voight-Kampff test. Uses .50 Kurtz.' Makes more sense.

Reply:
Actually, additional change as I forgot Mar'qua empire name changed, 

"A rare, custom-built revolver formerly used within Sol by the early Turing Police, gifted by the Mar'qua Technocracy. Use when there is no time for the Voight-Kampff test. Uses .50 Kurtz."